cli/config/configStruct.go

```diff
@@ -26,6 +27,7 @@ type ConfigStruct struct {
 	Auth                   configStructs.AuthConfig    `yaml:"auth"`
 	Config                 configStructs.ConfigConfig  `yaml:"config,omitempty"`
 	AgentImage             string                      `yaml:"agent-image,omitempty" readonly:""`
+	BasenineImage          string                      `yaml:"basenine-image,omitempty" readonly:""`
```

Comment:
what about kratos image?

Reply:
didn't add it to this place because we are focusing on the open source mizu now

Reply:
opened TRA-4201 for this